src/content/glossary/index.md

```diff
@@ -873,6 +873,10 @@ Short for "succinct non-interactive argument of knowledge", a SNARK is a type of
   Zero-knowledge rollups
 </DocLink>
 
+### Soft Fork {#soft-fork}
```

Comment:
Glossary convention is that everything should be lowercase unless it is an acronym or a proper noun.

```suggestion
### soft fork {#soft-fork}
```